src/rprblender/ui/render.py

```diff
@@ -196,6 +196,9 @@ def draw(self, context):
         if rpr.render_quality in ('LOW', 'MEDIUM', 'HIGH'):
             self.layout.prop(rpr, 'hybrid_low_mem')
 
+        if rpr.render_quality == 'FULL2':
+            self.layout.prop(rpr, texture_compression)
```

Comment:
use texture_compression in quotas '':
self.layout.prop(rpr, 'texture_compression')